qemu/tests/nested_test.py

```diff
@@ -60,7 +60,10 @@ def generate_parameter_file(params):
 
         guest_password = params.get("password")
 
+        bootstrap_options = params.get("bootstrap_options")
```

Comment:
@qcheng-redhat  After update cfg, here need be updated too: `params.get("nested_bs_options")`